app/src/androidTest/java/org/greenstand/android/TreeTracker/orgpicker/OrgPickerScreenKtTest.kt

```diff
@@ -0,0 +1,45 @@
+package org.greenstand.android.TreeTracker.orgpicker
+
+import androidx.compose.ui.test.*
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.greenstand.android.TreeTracker.activities.TreeTrackerActivity
+import org.greenstand.android.TreeTracker.models.organization.Org
+import org.junit.Assert.*
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.koin.androidx.compose.viewModel
+
+@RunWith(AndroidJUnit4::class)
+class OrgPickerScreenKtTest {
```

Comment:
This test only tests an org item. Can you also add test cases that test the rest of the screen?

Reply:
Ok. I am trying to test the **Select Organisation** text but it seems the string resource does not exist.